web/src/styles/app/form.css

```diff
@@ -33,3 +33,33 @@
 .select-condensed {
   height: 2rem;
 }
+
+.rdw-editor-main .DraftEditor-root {
+  min-height: unset; /* override of plugin */
+}
+
+/* add resizing capability to divs of react-draft-wysiwyg plugin */
+.public-DraftEditor-content {
+  resize: vertical;
```

Comment:
This gives us [drag-to-resize capability](https://developer.mozilla.org/en-US/docs/Web/CSS/resize) on the divs rendered by our RTE plugin. I didn't even know you could do this with pure CSS!